compiler/frontend/spark/src/main/java/edu/snu/nemo/compiler/frontend/spark/transform/ReduceTransform.java

```diff
@@ -99,7 +99,6 @@ public void onData(final T element) {
   @Override
   public void close() {
     // Write result to a temporary file.
-    // TODO #16: Implement collection of data from executor to client
```

Comment:
Why is this issue removed?

Reply:
Sorry for confusion, but this is the right issue.

Reply:
I've added it again. Thanks!